scripts/promote-suite

```diff
@@ -23,8 +23,8 @@ from typing import Tuple
 from debian import debfile
 
 BUILDER = Path(__name__).absolute().parent
-DEV_LFS = BUILDER.parent / 'securedrop-dev-packages-lfs'
-PROD_LFS = BUILDER.parent / 'securedrop-debian-packages-lfs'
+DEV_LFS = BUILDER.parent / 'securedrop-apt-test'
```

Comment:
Presumably this should be `TEST_LFS` now, as well as all the other variables named "dev". I'll submit a follow-up.